SS14.Launcher/Utility/ServerTagInfer.cs

```diff
@@ -48,7 +48,7 @@ public static IEnumerable<string> InferTags(ServerStatus status)
         // Infer 18+
         if (!currentTags.Contains(Tags.TagEighteenPlus))
         {
-            if (tagLikes.Contains("18+") || tagLikes.Contains("+18") || tagLikes.Contains("18"))
+            if (tagLikes.Contains("18+") || tagLikes.Contains("+18") || tagLikes.Contains("18") || tagLikes.Contains("18+.") || tagLikes.Contains("ERP"))
```

Comment:
Isn't +18. just doubling +18?

Reply:
> Isn't '18+.' just doubling '18+'?

No, launcher doesn't see the dots for some reason.

Reply:
`.Contains()` here is checking whether a collection contains the string, not whether a string contains the substring.